presto-product-tests/conf/tempto/tempto-configuration-for-docker-kerberos-simba.yaml

```diff
@@ -0,0 +1,13 @@
+databases:
```

Comment:
The `tempto-configuration-for-docker-kerberos-simba.yaml` and `tempto-configuration-for-docker-ldap-simba.yaml` are not referenced anywhere. Even if we do not have access to simba driver from OSS travis it would be nice to have a defnition in prestodb/master so as little as possible need to be redefined by user to run tests on simba drivers.

Reply:
There is a README file, how to use it.

I was thinking if it is a good idea to add them here, but I thought they may wors a documentation with real and working information how to configure simba jdbc with kerberos or ldap.

Though, I do not have strong opinion about this. I could add them to our infra instead. What do you think?

Reply:
I think it is nice to have them here. What I was suggesting, was that rather than removing those files, you could add more context related to them. A focused documentation explaining where to take driver from, and quoting specific commands which needs to be run to make use of those test-profiles, would be beneficial.

I think what we should opt for, is that person with some project knowledge, but not necessarily expert in the specific matter (product tests in this case) should be able to perform documented task (run product tests with simba drivers in this case).

PS. I am not suggesting that my PR fulfill that expectations very well, but yet I believe they should ;)